projects/angular/src/data/datagrid/_datagrid.clarity.scss

```
@@ -957,18 +1022,74 @@
     .datagrid-footer {
       min-height: tables-variables.$clr-table-compact-row-height;
       line-height: tables-variables.$clr-table-compact-row-height - tokens.$cds-global-space-1;
+      padding: 0 tokens.$cds-global-space-6;
 
       .pagination {
         line-height: tables-variables.$clr-table-compact-row-height;
+        gap: mixins.baselinePx(10);
 
-        .clr-page-size-select {
-          min-height: density.$clr-base-row-height-xs;
+        clr-dg-page-size {
+          gap: mixins.baselinePx(10);
+        }
+
+        .clr-select-wrapper {
+          max-height: tokens.$cds-global-space-9;
+          &::after {
+            --clr-forms-select-caret-size: #{tokens.$cds-global-space-6};
+            right: tokens.$cds-global-space-2;
+          }
+          .clr-page-size-select {
+            min-height: tokens.$cds-global-space-7;
+            padding: 0 mixins.baselinePx(20) 0 tokens.$cds-global-space-2;
+          }
         }
       }
 
       .clr-form-control-disabled .datagrid-footer-select.clr-checkbox-wrapper input[type='checkbox']:checked + label {
         top: 0;
       }
+
+      .column-switch-wrapper .column-toggle--action {
+        border-radius: tokens.$cds-global-space-2;
+        padding: 0 tokens.$cds-global-space-4;
+        height: mixins.baselinePx(20);
+      }
+    }
+
+    .pagination-list {
+      gap: tokens.$cds-global-space-4;
+
+      .pagination-current {
+        // @include mixins.generate-typography-token('CAPTION-LG-11');
+        padding: tokens.$cds-global-space-2 0;
+      }
+
+      .pagination-pages {
+        gap: tokens.$cds-global-space-2;
+      }
+
+      .pagination-current,
+      button {
+        @include mixins.min-equilateral(mixins.baselinePx(20));
```

Border radius is set to `var(--cds-alias-object-border-width-100)` in regular rule and `3px` in figma . 
Should have density token `clr-base-border-radius-s` in my opinion.

```suggestion
      button {
        border-radius: 2px
        @include mixins.min-equilateral(mixins.baselinePx(20));
```